### PR TITLE
Moving to SassDoc compliant comments

### PR DESCRIPTION
**Pull request by KittyGiraudel**

### CODE REVIEW

**Comment by kaelig** [comment]
Thanks a lot! Looks great.

For some reason I can't automatically merge the PR, can you make sure your fork was up to date?

**Comment by KittyGiraudel** [comment]
Should be better now. ;)

File: _rem.scss [hunk 1]
```diff
@@ -1,28 +1,55 @@
...
+ * @param {Number} $baseline ($guss-rem-baseline)
+ *
+ * @requires {variable} guss-rem-baseline
+ *
+ * @return {List | Number}
+ */
 @function rem($value, $baseline: $guss-rem-baseline) {
-    @if $value == 0 { @return 0; } // 0rem -> 0
+    @if $value == 0 {
+      @return 0; // 0rem -> 0
```
Comment (kaelig) [inline _rem.scss]: 4 space indentation

File: _rem.scss [hunk 2]
```diff
@@ -1,28 +1,55 @@
...
     }
 }
 
-
-// Output rem units with px fallback
-// Expects $properties to be a Sass map
-// Usage: http://sassmeister.com/gist/7451284
+/**
+ * Output `rem` units with `px` fallback.
+ * Expect `$properties` to be a map.
```
Comment (kaelig) [inline _rem.scss]: Not necessary with the `@param` (below) which is explicit enough

**Comment by KittyGiraudel** [comment]
All good.

File: _rem.scss [hunk 3]
```diff
@@ -1,28 +1,54 @@
...
+ * 
+ * @type Number
+ */
 $guss-rem-baseline: 10px !default;
 
-// Transform a value into rem
-// Assuming baseline is set to 10px on :root or html
+/**
+ * Transform a value into rem.
+ * Assuming baseline is set to `10px` on `:root` or `html`.
```
Comment (kaelig) [inline _rem.scss]: > Assuming baseline is set to `10px` on `:root` or `html`.

That's not true anymore, actually. I should have removed this line before.

File: _rem.scss [hunk 4]
```diff
@@ -1,28 +1,54 @@
...
+
         @each $e in $value {
             $result: append($result, rem($e, $baseline));
         }
+
         @return $result;
-    } @else {
+    } 
+
+    @else {
```
Comment (kaelig) [inline _rem.scss]: We use the `} @else {` notation, sorry :p

**Pull request merged**